### PR TITLE
AAP-14809-Revisions1: Resolved comments from google doc

### PR DESCRIPTION
**Pull request by lmalivert**
Jira link: [AAP-14809](https://issues.redhat.com/browse/AAP-14809)

This PR covers sections of Chapter 2 Hardening Ansible Automation Platform.
Please review it against this [google doc](https://docs.google.com/document/d/1A6VLRNEzvKw4GWhcQpIMpSN9ylFpMGsGLeIUU-7p19I/edit#heading=h.a2af4bqois8b).

**Technical reviewers (SMEs) please review the instructions in**:
- proc-configure-centralized-logging
- proc-controller-stig-considerations
- proc-file-systems-mounted-noexec

**Notes for peer-reviewer(s)**
- ref-dns-load-balancing: DNS and load balancing title is not showing up, but the content is. I can't figure out why it's doing this.

### CODE REVIEW

**Review by emurtoug — APPROVED** [review]
LGTM

File: downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc [hunk 1]
```diff
@@ -9,10 +9,13 @@
...
 
 When planning firewall or cloud network security group configurations related to {PlatformNameShort}, see the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_planning_guide/index#ref-network-ports-protocols_planning[Network ports and protocols] section of the {PlatformNameShort} Planning Guide to understand what network ports need to be opened on a firewall or security group.
 
+For more information on using a load balancer, and for outgoing traffic requirements for services compatible with {PlatformNameShort}. Consult the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6756251[What ports need to be opened in the firewall for Ansible Automation Platform 2 Services?]. For nternet-connected systems, this article also defines the outgoing traffic requirements for services that {PlatformNameShort} can be configured to use, such as Red Hat Automation Hub, Red Hat Insights for AAP, Ansible Galaxy, the registry.redhat.io container image registry, and so on.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc:17]: Edit "nternet-connected systems" to "internet-connected systems"

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
+Visibility and analytics is an important pillar of Enterprise Security and Zero Trust Architecture. Logging is key to capturing actions and auditing. This can be managed through the built-in audit support by following the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/auditing-the-system_security-hardening[Auditing the system] section of the Red Hat Enterprise Linux Security and Hardening Guide. Controller's built-in logging and activity stream support logs all changes within controller and automation logs for auditing purposes. More detailed information is available in the link:https://docs.ansible.com/automation-controller/latest/html/administration/logging.html[Logging and Aggregation] section of the {ControllerName} documenation.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-logging-log-capture.adoc:10]: Rephrase and add attributes for controller

"Built-in logging and activity stream support for {ControllerName} logs all changes within {ControllerName} and automation logs for auditing purposes."

File: downstream/modules/aap-hardening/con-install-secure-host.adoc [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
+The installation host must be a Red Hat Enterprise Linux server that has been installed and configured in accordance with link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/index[RHEL hardening guidance] and any security profile requirements relevant to your organization (CIS, STIG, and so on). Obtain the AAP installer as described in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_planning_guide/index#choosing_and_obtaining_a_red_hat_ansible_automation_platform_installer[Automation Platform Planning Guide], and create the installer inventory file as describe in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Automation Platform Installation Guide]. This inventory file is used for upgrades, adding infrastructure components, and day-two operations by the installer, so preserve the file after installation for future operational use.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-install-secure-host.adoc:12]: Edit link title "[RHEL hardening guidance]" to "[Security hardening for Red Hat Enterprise Linux]"

File: downstream/modules/aap-hardening/ref-security-variables-install-inventory.adoc [hunk 4]
```diff
@@ -24,12 +24,12 @@ The following table lists a number of security-relevant variables and their reco
...
 
 [NOTE]
 ====
-This guide recommends using an external database in production environments, but for development and testing scenarios the database could be co-located on the controller. Due to current PostgreSQL 13 limitations, setting `pg_sslmode = verify-full` when the database is co-located on the controller results in an error validating the host name during TLS negotiation.  Until this issue is resolved, an external database must be used in order to ensure mutual TLS authentication between the controller and the database.
+This guide recommends using an external database in production environments, but for development and testing scenarios the database could be co-located on the controller. Due to current PostgreSQL 13 limitations, setting `pg_sslmode = verify-full` when the database is co-located on the controller results in an error validating the host name during TLS negotiation. Until this issue is resolved, an external database must be used to ensure mutual TLS authentication between the controller and the database.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/ref-security-variables-install-inventory.adoc:34]: Add controller attribute x2

"the database could be co-located on {ControllerName}."

"authentication between {ControllerName} and the database."

File: downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc [hunk 5]
```diff
@@ -7,9 +7,9 @@
...
 
-An important aspect of backups is that they contain a copy of the database as well as the secret key used to decrypt credentials stored in the database, so the backup files should be stored in a secure encrypted location. This means that access to endpoint credentials are protected properly. Access to backups should be limited only to {PlatformNameShort} administrators who have root shell access to the controller.
+An important aspect of backups is that they contain a copy of the database as well as the secret key used to decrypt credentials stored in the database, so the backup files should be stored in a secure encrypted location. This means that access to endpoint credentials are protected properly. Access to backups should be limited only to {PlatformNameShort} administrators who have root shell access to the controller and th dedicated installation host.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc:12]: Add controller attribute and edit typo ("th")

"root shell access to {ControllerName} and the dedicated installation host."

**Review by emurtoug — CHANGES_REQUESTED** [review]

File: downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc [hunk 6]
```diff
@@ -7,9 +7,9 @@
 
 [role="_abstract"]
 
-The contingency planning security control indicates that information system backups reflect the requirements in contingency plans and other organizational requirements for backing up information. Recommendations exist for High Availability and Backup complex operations and architectures. We will cover them briefly here, but they are outside the scope of the {PlatformNameShort} STIG. As the {PlatformNameShort} STIG indicates, the operations normally require significant planning, people, and processes outside a single product.
+Taking regular backups of Ansible Automation Platform is a critical part of disaster recovery planning.  Both backups and restores are performed using the installer, so these actions should be performed from the dedicated installation host described earlier in this document.  Refer to the link:https://docs.ansible.com/automation-controller/latest/html/administration/backup_restore.html[Backing Up and Restoring]section of the automation controller documentation for further details on how to perform these operations.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc:10]: Delete extra spaces: 

"planning. Both backups"
"document. Refer to"

File: downstream/modules/aap-hardening/con-rhel-host-planning.adoc [hunk 7]
```diff
@@ -3,10 +3,10 @@
...
+The security of {PlatformNameShort} relies in part on the configuration of the underlying Red Hat Enterprise Linux servers. For this reason, the underlying Red Hat Enterprise Linux hosts for each {PlatformNameShort} component must be installed and configured in accordance with the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index[RHEL 8 Security Hardening Guide] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/security_hardening/index[RHEL 9 Security Hardening Guide] (depending on which operating system will be used), as well as any security profile requirements (CIS, STIG, HIPAA, and so on) used by your organization.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-rhel-host-planning.adoc:10]: Edit link titles: 
"[RHEL 8 Security Hardening Guide]" to "[Security hardening for Red Hat Enterprise Linux 8]"
"[RHEL 9 Security Hardening Guide]" to "[Security hardening for Red Hat Enterprise Linux 9]"

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc (hunk 2 above)
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-logging-log-capture.adoc:10]: Edit link title "Red Hat Enterprise Linux Security and Hardening Guide" to "Security hardening for Red Hat Enterprise Linux"

File: downstream/modules/aap-hardening/con-rhel-host-planning.adoc [hunk 8]
```diff
@@ -3,10 +3,10 @@
...
 
-Note that applying certain security controls from the STIG or other security profiles may conflict with {PlatformNameShort} support requirements. Some examples are listed in the xref:<link>[<link_]{ControllerNameStart} STIG considerations] section, although this is not an exhaustive list. In order to maintain a supported configuration, be sure to discuss any such conflicts with your security auditors so the AAP requirements are understood and approved.
+Note that applying certain security controls from the STIG or other security profiles may conflict with {PlatformNameShort} support requirements. Some examples are listed in the xref:<link>[<link_]{ControllerNameStart} STIG considerations] section, although this is not an exhaustive list. In order to maintain a supported configuration, be sure to discuss any such conflicts with your security auditors so the {PlatformNameShort} requirements are understood and approved.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/con-rhel-host-planning.adoc:12]: Add xref

"xref:con-controller-stig-considerations.adoc[{ControllerNameStart} STIG considerations]"

File: downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc [hunk 9]
```diff
@@ -25,7 +25,12 @@ The following Ansible content collections are available for managing AAP compone
...
+These practices allow version control of all infrastructure and configuration. Apply Git best practices to ensure proper code quality inspection prior to being synchronized into {PlatformNameShort}. Relevant Git best practices include the following: 
 
-IaC also encourages using external vault systems (see xref:<link>[<link_External credential vault considerations] which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed.
+* Creating pull requests.
+* Ensuring that inspection tools are in place.
+* Ensuring that no plain text secrets are committed.
+* Ensuring that pre-commit hooks and any other policies are followed.
+
+IaC also encourages using external vault systems which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed. For more information on using external vault systems, see section link:<internal link>[2.3.2.3 External credential vault considerations] within this guide.
```
Comment (emurtoug) [inline downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc:35]: Add xref

"xref:con-external-credential-vault.adoc[2.3.2.3 External credential vault considerations]"

**Review by emurtoug — CHANGES_REQUESTED** [review]

File: downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc [hunk 10]
```diff
@@ -41,7 +41,7 @@ If this field is not set to "/var/log/awx", then follow these configuration step
 + 
 Note that this change will need to be made on each {ControllerName} in a load-balanced scenario.
 
-All user session data must be logged in order to support the troubleshooting, debugging and forensic analysis for visibility and analytics. Without this data from the controller’s web server, important auditing and analysis for event investigations will be lost. To modify or verify this is configured in the system, follow these steps:
+All user session data must be logged to support troubleshooting, debugging and forensic analysis for visibility and analytics. Without this data from the controller’s web server, important auditing and analysis for event investigations will be lost. To modify or verify that the system is configured to ensure that user session data is logged, use the following steps:
```
Comment (demonpig) [inline downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc:44]: "To modify or verify that the system is configure to ensure that user session data is logged" reads a little weird to me. Maybe just rephrase it to "To verify that the system...". Essentially just remove the "modify or" from that sentence.

File: downstream/modules/aap-hardening/proc-configure-external-authentication.adoc [hunk 11]
```diff
@@ -7,7 +7,7 @@
 
 [role="_abstract"]
 
-As noted in the link:(add external link) external authentication is recommended for user access to the automation controller. After you choose the authentication type that best suits your needs, navigate to menu:Settings[Authentication] in the automation controller UI, click on the relevant link for your authentication back-end, and follow the relevant instructions for link:https://docs.ansible.com/automation-controller/latest/html/administration/configure_tower_in_tower.html#authentication[configuring the authentication] connection.
+As noted in the link:(add internal link) external authentication is recommended for user access to the automation controller. After you choose the authentication type that best suits your needs, navigate to menu:Settings[Authentication] in the automation controller UI, click on the relevant link for your authentication back-end, and follow the relevant instructions for link:https://docs.ansible.com/automation-controller/latest/html/administration/configure_tower_in_tower.html#authentication[configuring the authentication] connection.
```
Comment (demonpig) [inline downstream/modules/aap-hardening/proc-configure-external-authentication.adoc:10]: do we need to add an internal link here?

File: downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc (hunk 10 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc:44]: Done. Thank you!

File: downstream/modules/aap-hardening/proc-configure-external-authentication.adoc (hunk 11 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/proc-configure-external-authentication.adoc:10]: Internal link added.

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc [hunk 12]
```diff
@@ -7,8 +7,8 @@
...
+Another critical capability of logging is the ability to use cryptography to protect the integrity of log tools. Log data includes all information (e.g., log records, log settings, and log reports) needed to successfully log information system activity. It is common for attackers to replace the log tools or inject code into the existing tools to hide or erase system activity from the logs. To address this risk, log tools must be cryptographically signed so that you can identify when the log tools have been modified, manipulated, or replaced. For example, one way to validate that the log tool(s) have not been modified, manipulated or replaced is to use a checksum hash against the tool file(s). This ensures the integrity of the  tool(s) have not been compromised.
```
[the remark this answers is not in the document]
Comment (lmalivert) [reply downstream/modules/aap-hardening/con-logging-log-capture.adoc:14]: Done.

File: downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc (hunk 5 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/proc-disaster-recovery-operations.adoc:12]: Done

File: downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc [hunk 13]
```diff
@@ -9,14 +9,17 @@
...
 
 When planning firewall or cloud network security group configurations related to {PlatformNameShort}, see the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_planning_guide/index#ref-network-ports-protocols_planning[Network ports and protocols] section of the {PlatformNameShort} Planning Guide to understand what network ports need to be opened on a firewall or security group.
 
+For more information on using a load balancer, and for outgoing traffic requirements for services compatible with {PlatformNameShort}. Consult the Red Hat Knowledgebase article link:https://access.redhat.com/solutions/6756251[What ports need to be opened in the firewall for Ansible Automation Platform 2 Services?]. For internet-connected systems, this article also defines the outgoing traffic requirements for services that {PlatformNameShort} can be configured to use, such as {HubNameMain}, {InsightsName}, {Galaxy}, the registry.redhat.io container image registry, and so on.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc:17]: Should registry.redhat.io be enclosed in tick marks here? `registry.redhat.io`

File: downstream/modules/aap-hardening/con-rhel-host-planning.adoc [hunk 14]
```diff
@@ -3,10 +3,10 @@
...
 
-Note that applying certain security controls from the STIG or other security profiles may conflict with {PlatformNameShort} support requirements. Some examples are listed in the xref:<link>[<link_]{ControllerNameStart} STIG considerations] section, although this is not an exhaustive list. In order to maintain a supported configuration, be sure to discuss any such conflicts with your security auditors so the AAP requirements are understood and approved.
+Note that applying certain security controls from the STIG or other security profiles may conflict with {PlatformNameShort} support requirements. Some examples are listed in the xref:con-controller-stig-considerations.adoc[{ControllerNameStart} STIG considerations] section, although this is not an exhaustive list. In order to maintain a supported configuration, be sure to discuss any such conflicts with your security auditors so the {PlatformNameShort} requirements are understood and approved.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-rhel-host-planning.adoc:12]: Suggest rewording "In order to maintain..." to "To maintain..." to make it more concise.

File: downstream/modules/aap-hardening/proc-file-systems-mounted-noexec.adoc [hunk 15]
```diff
@@ -7,13 +7,20 @@
...
+// These steps are rewritten below. Waiting on SME to verify before deleting this paragraph: These file systems must be re-mounted with the "noexec" option removed in order to install {PlatformNameShort}. Once installation is complete, the "noexec" option can be reapplied to the `/tmp` and `/var/tmp` file systems. When this is completed, change the {ControllerName}'s job execution path from `/tmp` to an alternate directory that does not have the "noexec" option enabled. To make this change, log in to the {ControllerName} UI as an administrator, and navigate to "Settings > Jobs settings", then change the "Job execution path" setting to the alternate directory.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/proc-file-systems-mounted-noexec.adoc:16]: Suggest removing "...in order to..." to make sentence more concise.

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc [hunk 16]
```diff
@@ -7,8 +7,8 @@
...
+Another critical capability of logging is the ability to use cryptography to protect the integrity of log tools. Log data includes all information (for example, log records, log settings, and log reports) needed to successfully log information system activity. It is common for attackers to replace the log tools or inject code into the existing tools to hide or erase system activity from the logs. To address this risk, log tools must be cryptographically signed so that you can identify when the log tools have been modified, manipulated, or replaced. For example, one way to validate that the log tool(s) have not been modified, manipulated or replaced is to use a checksum hash against the tool file(s). This ensures the integrity of the  tool(s) have not been compromised.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-logging-log-capture.adoc:14]: Suggest rewriting "This ensures the integrity of the tool(s) **has** not been compromised." It refers to the integrity (singular) and not tools (plural).

File: downstream/modules/aap-hardening/con-platform-components.adoc [hunk 17]
```diff
@@ -3,7 +3,7 @@
 
 [id="con-platform-components_{context}"]
 
-= {PlatformName} components
+= {PlatformNameShort} components
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-platform-components.adoc:7]: Noticed link in line 14 of this file points to the wrong id and should be updated to:
link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_planning_guide/ref-aap-components

File: downstream/modules/aap-hardening/con-rhel-host-planning.adoc (hunk 14 above)
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-rhel-host-planning.adoc:12]: Suggest rewording to remove the phrase "In order to...". "To maintain a supported configuration..." is more concise.

File: downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc [hunk 18]
```diff
@@ -80,7 +80,7 @@ To set up logging to any of the aggregator types, read the documentation on the
 . When done, click *Save* to apply the settings or Cancel to abandon the changes.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc:80]: I think these are buttons and should be tagged as "Click btn:[Save]" or "btn:[Cancel]". At a minimum the UI selections in this section should be bolded.

File: downstream/modules/aap-hardening/proc-file-systems-mounted-noexec.adoc [hunk 19]
```diff
@@ -7,13 +7,20 @@
 
 [role="_abstract"]
 
-The Red Hat Enterprise Linux (RHEL) 8 STIG requires that a number of file systems are mounted with the "noexec" option to prevent execution of binaries located in these file systems. The {PlatformNameShort} (AAP) installer runs a preflight check that will fail if any of the following file systems are mounted with the "noexec" option:
+The Red Hat Enterprise Linux 8 STIG requires that a number of file systems are mounted with the "noexec" option to prevent execution of binaries located in these file systems. The {PlatformNameShort} installer runs a preflight check that will fail if any of the following file systems are mounted with the "noexec" option:
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/proc-file-systems-mounted-noexec.adoc:10]: Should noexec be enclosed in single ticks instead of quotes to render as a code phrase?

File: downstream/modules/aap-hardening/ref-architecture.adoc [hunk 20]
```diff
@@ -3,14 +3,15 @@
...
 
 [role="_abstract"]
 
-For large-scale production environments with availability requirements, this guide recommends deploying the above-mentioned components as described in the link:https://access.redhat.com/documentation/en-us/reference_architectures/2021/html-single/deploying_ansible_automation_platform_2.1/index#overview[reference architecture] documentation for {PlatformName} on RHEL. While some variation may make sense for your specific technical requirements, following the reference architecture results in a supported production-ready environment.
+For large-scale production environments with availability requirements, this guide recommends deploying the components described in section 2.1 of this guide using the instructions in the link:https://access.redhat.com/documentation/en-us/reference_architectures/2021/html-single/deploying_ansible_automation_platform_2.1/index#overview[reference architecture] documentation for {PlatformName} on Red Hat Enterprise Linux. While some variation may make sense for your specific technical requirements, following the reference architecture results in a supported production-ready environment.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/ref-architecture.adoc:10]: Can an xref be added for section 2.1 here?

File: downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc [hunk 21]
```diff
@@ -23,9 +23,14 @@ The following Ansible content collections are available for managing AAP compone
...
+These practices allow version control of all infrastructure and configuration. Apply Git best practices to ensure proper code quality inspection prior to being synchronized into {PlatformNameShort}. Relevant Git best practices include the following: 
 
-IaC also encourages using external vault systems (see xref:<link>[<link_External credential vault considerations] which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed.
+* Creating pull requests.
+* Ensuring that inspection tools are in place.
+* Ensuring that no plain text secrets are committed.
+* Ensuring that pre-commit hooks and any other policies are followed.
+
+IaC also encourages using external vault systems which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed. For more information on using external vault systems, see section link:con-external-credential-vault.adoc[2.3.2.3 External credential vault considerations] within this guide.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc:35]: Should this be spelled out? Infrastructure as Code (IaC) or was it previously defined earlier?

**Review by dcdacosta — COMMENTED** [review]
A few minor corrections needed; otherwise, it looks good!

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc [hunk 22]
```diff
@@ -7,8 +7,8 @@
...
+Visibility and analytics is an important pillar of Enterprise Security and Zero Trust Architecture. Logging is key to capturing actions and auditing. You can manage logging and auditing by using the built-in audit support described in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/auditing-the-system_security-hardening[Auditing the system] section of the Security hardening for Red Hat Enterprise Linux guide. Controller's built-in logging and activity stream support {ControllerName} logs all changes within {ControllerName} and automation logs for auditing purposes. More detailed information is available in the link:https://docs.ansible.com/automation-controller/latest/html/administration/logging.html[Logging and Aggregation] section of the {ControllerName} documenation.
```
Comment (dcdacosta) [inline downstream/modules/aap-hardening/con-logging-log-capture.adoc:10]: typo: documenation should be documentation.

File: downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc [hunk 23]
```diff
@@ -31,7 +31,7 @@ awx-manage print_settings LOG_AGGREGATOR_MAX_DISK_USAGE_PATH
 If this field is not set to "/var/log/awx", then follow these configuration steps: 
 +
 --
-.. Open a web browser and navigate to: link:https://<Automation Controller server>/api/v2/settings/logging/ (If the "Log In" button is displayed, click it, enter an Automation Controller administrator's credentials, and continue).
+.. Open a web browser and navigate to: link:https://<{ControllerName} server>/api/v2/settings/logging/ (If the "Log In" button is displayed, click it, enter an Automation Controller administrator's credentials, and continue).
```
Comment (jce-redhat) [inline downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc:34]: i'd change this to "Open a web browser and navigate to https://<FQDN>/api/v2/settings/logging/, where <FQDN> is the fully-qualified hostname of your automation controller.  If the "Log In" button is displayed, click it, log in as an automation controller adminstrator account, and continue."

File: downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc (hunk 13 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc:17]: No, it should not.

File: downstream/modules/aap-hardening/con-rhel-host-planning.adoc (hunk 14 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/con-rhel-host-planning.adoc:12]: Done.

File: downstream/modules/aap-hardening/con-logging-log-capture.adoc (hunk 16 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/con-logging-log-capture.adoc:14]: Done

File: downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc (hunk 21 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc:35]: Yes, it was previously defined in line ten.

File: downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc (hunk 23 above)
Comment (lmalivert) [reply downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc:34]: Done.

**Comment by lmalivert** [comment]
Hi @dcdacosta and @jce-redhat! Thank you for your review. I made all of the suggested edits.

**Review by dcdacosta — APPROVED** [review]
Awesome job! LGTM!
